server/sonar-web/src/main/js/app/components/nav/global/GlobalNav.tsx

```diff
@@ -158,4 +158,6 @@ const mapStateToProps = state => {
 
 const mapDispatchToProps = { skipOnboarding };
 
-export default connect(mapStateToProps, mapDispatchToProps)(GlobalNav);
+export default connect<StateProps, DispatchProps, {}>(mapStateToProps, mapDispatchToProps)(
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This is the default value for this type parameter, so it can be omitted. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS4157)